testerum-backend/testerum-runner/testerum-runner-cmdline/src/main/kotlin/com/testerum/runner_cmdline/runner_tree/nodes/feature/RunnerFeature.kt

```diff
@@ -12,11 +9,14 @@ import com.testerum.runner_cmdline.runner_tree.nodes.RunnerFeatureOrTest
 import com.testerum.runner_cmdline.runner_tree.nodes.RunnerTreeNode
 import com.testerum.runner_cmdline.runner_tree.runner_context.RunnerContext
 import com.testerum.runner_cmdline.runner_tree.vars_context.GlobalVariablesContext
+import com.testerum_api.testerum_steps_api.test_context.ExecutionStatus
+import com.testerum_api.testerum_steps_api.test_context.ExecutionStatus.PASSED
+import com.testerum_api.testerum_steps_api.test_context.ExecutionStatus.SKIPPED
 
 class RunnerFeature(featurePathFromRoot: List<String>,
-                    private val featureName: String,
-                    private val tags: List<String>,
-                    private val featuresOrTests: List<RunnerFeatureOrTest>,
+                    val featureName: String,
+                    val tags: List<String>,
+                    val featuresOrTests: List<RunnerFeatureOrTest>,
```

Comment:
Keep the fields private, and add a calculated ``title`` property:
```kotlin
val title: String
    get() = featureName
```

Do this also for ``RunnerParametrizedTest``, ``RunnerScenario`` (including keeping the ``scenarioName`` private), ``RunnerSuite``, and ``RunnerTest``.